android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/login/LoginPicker.kt

```diff
@@ -54,6 +58,8 @@ internal class LoginPicker(
 
     @Suppress("TooGenericExceptionCaught")
     fun dismissCurrentLoginSelect(promptRequest: PromptRequest.SelectLoginPrompt? = null) {
+        emitLoginAutofillDismissedFact()
```

Comment:
🔍 nit: this one is not above `idePrompt()` like the other ones 😅

Reply:
d'oh! moved this one below the `try-catch` block